src/interceptors/request.ts

```diff
@@ -58,6 +59,54 @@ export function defaultRequestInterceptor(axios: AxiosCacheInstance): RequestInt
       }
     }
 
+    // whitelistedUrls
+    if (
+      typeof config.cache.cachePredicate === 'object' &&
+      config.cache.cachePredicate.whitelistedUrls &&
+      config.url
+    ) {
+      let matched = false;
+
+      for (const url of config.cache.cachePredicate.whitelistedUrls) {
+        if (
+          url instanceof RegExp
+            ? // Handles stateful regexes
+              // biome-ignore lint: reduces the number of checks
+              ((url.lastIndex = 0), url.test(config.url))
+            : config.url.includes(url)
+        ) {
+          matched = true;
+
+          if (__ACI_DEV__) {
+            axios.debug({
+              id: config.id,
+              msg: `Cached because url (${config.url}) matches whitelistedUrls (${config.cache.cachePredicate.whitelistedUrls})`,
```

Comment:
pretty sure you can extract both these logs and the loop into a generic function and just call both for allow and ignore.